development/development/html/guidance.rst

```diff
@@ -0,0 +1,140 @@
+Idiomatic HTMLGuidelines
+================================================
+
+1. Whitespace
+----------------
+
+Only one style should exist across the entire source of your code-base.
+Always be consistent in your use of whitespace. Use whitespace to
+improve readability.
+
+-  Never mix spaces and tabs for indentation.
+-  Preference: ``Tabs``
+
+.. tip: configure your editor to “show invisibles”. This will allow you to
+eliminate end of line whitespace, eliminate unintended blank line
+whitespace, and avoid polluting commits.
+
+2. Format
+------------
+
+-  Always use lowercase tag and attribute names.
+-  Write one discrete element per line.
+-  Use one additional level of indentation for each nested element.
+-  Use valueless boolean attributes (e.g. ``checked`` rather than
+   ``checked="checked"``).
+-  Always use double quotes to quote attribute values.
+-  Omit the ``type`` attributes from ``link`` stylesheet, ``style`` and
+   ``script`` elements.
+-  Always include closing tags. ``<p>Some text</p>`` **NOT** ``<p>Some text``
+-  Don’t include a trailing slash in self-closing elements.
```

Comment:
```suggestion
-  Don’t include a trailing slash in self-closing elements, e.g. ``<br>`` instead of ``<br/>``
```

Reply:
thats a bad example lol. We do not want them to use breaks as they are not styleable

Reply:
ok :D